### ISSUE TITLE
Add network connection test button to console

### ISSUE THREAD

**Issue by yvonnewilson** [auth0/ad-ldap-connector#82]
Add to ad/ldap connector console a button to test network connectivity to Auth0 server from AD/LDAP connector so that when customer claims their network is fine, the test can be easily done to prove/disprove this.

**Comment by jholladay10** [comment]
Are you suggesting that it perform the full websocket connection here, or that it perform some other type of network diagnostics?  Running the server from the command line will quickly fail if it can't make the connection.  If running as a service, it will continuously fail and restart.  Do you think that more info in the log would suffice?

**Comment by yvonnewilson** [comment]
Yes, info in log file would suffice. It would help if it was able to make a distinction in logs between a network error (cannot get to Auth0 at all) vs able to reach Auth0, but some issue on the Auth0 side prevents the connection from being established.
This was filed almost 3 years ago - sometimes there were issues with Auth0 server that prevented connections and sometimes the customer network had blocked connections, and it was hard to tell the difference from the logs at the time.
